aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/DataSetupExtension.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.ccd.definitionstore.tests;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.apache.commons.lang.BooleanUtils;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import uk.gov.hmcts.ccd.definitionstore.tests.helper.idam.AuthenticatedUser;
+
+import java.io.File;
+
+import static org.hamcrest.CoreMatchers.anyOf;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.extension.ExtensionContext.Namespace.GLOBAL;
+
+/**
+ * Custom Extension which executes setup code only once before all tests are started.
+ * This is temporary solution until https://github.com/junit-team/junit5/issues/456 released
+ */
+public class DataSetupExtension implements BeforeAllCallback {
+
+    static final String IMPORTED = "imported";
+
+    @Override
+    public void beforeAll(ExtensionContext context) throws Exception {
+        if (!BooleanUtils.isTrue(context.getRoot().getStore(GLOBAL).get(IMPORTED, Boolean.class))) {
```

Comment:
shall we rename `IMPORTED` to a more general `INITIALISED` or something sim? because we are not only doing import but also setting up roles